### DISCUSSION TITLE
2020/12/10/datadog-terraform/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#181] [Blog Comments]
# 2020/12/10/datadog-terraform/

여러 클라우드 환경에서 이곳저곳 산재되어 있는 클라우드 리소스들을 통합적으로 모니터링이 가능하게 해주는 모니터링 솔루션인 Datadog을 알아 보자. Dat

https://tech.cloudmt.co.kr/2020/12/10/datadog-terraform/

**Comment by kilkat** [comment]
좋은 글 잘 봤습니다. 한가지 궁금한점이 있어서 댓글 드립니다. api key만 있어도 datadog에 데이터 전송이 가능할텐데(python를 사용해도 api key만 명시해도 데이터 전송이 가능함) application key까지 명시하는 이유는 뭘까요? 현재 제가 api key와 application key의 역할에 대한 이해도가 명확하지 않아서 도움 주시면 감사하겠습니다.
